src/super_gradients/training/params.py

```diff
@@ -98,7 +99,7 @@
         # "lr_updates": {"type": "array", "minItems": 1},
         "lr_decay_factor": {"type": "number", "minimum": 0, "maximum": 1},
         "lr_warmup_epochs": {"type": "number", "minimum": 0, "maximum": 10},
-        "initial_lr": {"type": "number", "exclusiveMinimum": 0, "maximum": 10},
+        # "initial_lr": {"type": "number", "exclusiveMinimum": 0, "maximum": 10},
```

Comment:
If you want to support number | Dict[str,number]

```
"anyOf": [
    {
      "type": ["number", "string", "boolean", "null"]
    },
    {
      "type": "object",
      "patternProperties": {
        "^[a-zA-Z0-9_\.]+$": { "type": "number" }
      },
      "additionalProperties": false
    }
  ]
```

Reply:
fixed https://github.com/Deci-AI/super-gradients/commit/966180d62ef7e8c1a5eb1e2a98a2f9a8783c9af3